python/lsst/ts/atmcssimulator/schemas/logevent_allAxesInPosition.json

```diff
@@ -3,7 +3,7 @@
     "type": "object",
     "properties": {
         "id": {
-            "const": "allAxesInPosition"
+            "const": "evt_allAxesInPosition"
```

Comment:
In the long term I hope you will make these schemas into normal classes (much like the commands I suggested above). Advantages include:
* It is checked at import time.
* It is more readable and reduces boilerplate.
* It avoids the hassle of translating between json and python.
* It eliminates a language.

Reply:
I understand all those advantages. The advantages of keeping them as JSON files is 

- They can easily be used in documentation
- IDEs can highlight the elements and indicate errors
- LabVIEW and other non-Python code can use them

In other words, there are advantages to both.

Reply:
I started this way with saving CSC config schema in files and we ended up deciding we were happier having them in code. I admit it had one additional advantage that might not apply here: it simplified unit tests of schema files. But I think the other reasons were just as important. For docs we give a link to the github code file.

Reply:
After our chat on Slack I opened DM-39802 for this.